src/calculateRentalCost.js

```diff
@@ -5,6 +5,22 @@
  */
 function calculateRentalCost(days) {
   // write code here
+
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+  const SHORT_TERM = 3;
+  const SHORT_TERM_DISCOUNT = 20;
+  const PAY_FOR_DAY = 40;
+
+  if (days >= LONG_TERM) {
+    return days * PAY_FOR_DAY - LONG_TERM_DISCOUNT;
+  }
+
+  if (days >= SHORT_TERM) {
+    return days * PAY_FOR_DAY - SHORT_TERM_DISCOUNT;
+  }
```

Comment:
The current implementation applies the long-term discount first, which is correct. However, if the number of days is exactly 7, both conditions (long-term and short-term) will be true. The current order ensures the long-term discount is applied, but it's important to ensure this logic is intentional and aligns with the task requirements. If the task requires different behavior, adjust the conditions accordingly.